.eslintrc.js

```diff
@@ -1,33 +1,34 @@
 module.exports = {
-	env: {
-		es6: true,
-		node: true,
-	},
-	extends: ['eslint:recommended', 'plugin:react/recommended', 'plugin:@typescript-eslint/eslint-recommended'],
-	parserOptions: {
-		ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
-		sourceType: 'module', // Allows for the use of imports
-		ecmaFeatures: {
-			jsx: true, // Allows for the parsing of JSX
-		},
-	},
-	parser: '@typescript-eslint/parser',
-	plugins: ['react', '@typescript-eslint', 'testing-library'],
-	rules: {
-		'@typescript-eslint/ban-ts-ignore': 'off',
-		'@typescript-eslint/no-explicit-any': 'off',
-		'no-unused-vars': 'off',
-		'@typescript-eslint/no-unused-vars': [
-			2,
-			{
-				args: 'none',
-			},
-		],
-		'react/prop-types': 0,
-	},
-	settings: {
-		react: {
-			version: 'detect', // Tells eslint-plugin-react to automatically detect the version of React to use
-		},
-	},
-};
+  env: {
```

Comment:
Can you make sure you respect .editorconfig in your IDE? Or did the storybook ESLint install mess with this?
Can you open and re-save in that case?